Travel Scheduler/Helper Files/TravelSchedulerHelper.m

```diff
@@ -142,6 +142,15 @@ float getMin(float num1, float num2)
     return (num1 > num2) ? num2 : num1;
 }
 
+void getDistanceToHome(Place *place, Place *home)
+{
+    Commute *commuteInfo = [[Commute alloc] initWithOrigin:place.placeId toDestination:home.placeId withDepartureTime:0];
+    commuteInfo.origin = place;
+    commuteInfo.destination = home;
```

Comment:
Wait, doesn't the init method from commute already do that? If not I think it is better to do that there (I will check and change it if you guys agree)

Reply:
No, I don't think so? The init method only makes an API request that calculates the distances (time, steps, duration...) but doesn't give any information about the places from what I can see.